File: java/pe/src/unittest/Pe2Test.java
package unittest;

import static org.junit.Assert.*;
import static org.hamcrest.CoreMatchers.*;
import ktn.pe.Pe2;

import org.junit.Test;

public class Pe2Test {

    @Test
    public void test() {
        Pe2 pe2 = new Pe2();
        
        assertThat("0 2", pe2.pe2(0L), is(2L));
        assertThat("8 10", pe2.pe2(8L), is(10L));
    }

}

File: kt/pe/src/test/kotlin/ktn/pe/Pe2Test.kt
package ktn.pe

import org.junit.Assert.*
import org.hamcrest.CoreMatchers.*

import org.junit.Test

class Pe2Test {

    @Test
    fun test() {
        val pe2 = Pe2()

        assertThat("0 2", pe2.pe2(0L), `is`(2L))
        assertThat("8 10", pe2.pe2(8L), `is`(10L))
    }

}
